docs/INSTALL.md

```diff
@@ -172,66 +172,71 @@ In this case the filesystem will be mounted at startup, and the sfs process will
 Sync daemon component
 ===============
 
-Configuring php-sync on the central node
+Prerequisites
+--------------------
+Install php, if not already present.
+
+Make sure you have installed php together with:
+- php-sysvmsg
+- php-sysvsem
+- php-sysvshm
+
+Check the configuration directive `disable_functions` of `/etc/php.ini` for occurences of
+- proc_open
+- proc_close
+- proc_get_status
+- proc_nice
+- proc_terminate
+- posix_getpid
+- system
+
+
+Installation
 --------------------
 
 In this setup we're going to run the sync daemon only on the central node.
+You can run the sync daemon using rsync over ssh or use the plain rsync transfer in trusted environments.
+
+First copy the file `sfs-sync.php` into the directory `/usr/local/bin`.
```

Comment:
I don't want these kind of assumptions in the documentation, sorry. You don't need to copy `sfs-sync.php` to any other directory in order to make it work.